File: nomad-samples/admin/js/event-instance-scheduler/event-instance/delete-instance.js
import * as prjConstants from "../constants/project-constants.js";
import apiExceptionHandler from "../exceptions/api-exception-handler.js";

export default async function deletingEventInstance(AUTH_TOKEN, ID, CONTENT_DEFINITION_ID) 
{
		// Create header for the request
    const HEADERS = new Headers();
    HEADERS.append("Content-Type", "application/json");
    HEADERS.append("Authorization", `Bearer ${AUTH_TOKEN}`);

    // Post
    const RESPONSE = await fetch(`${prjConstants.ADMIN_API_URL}/content/${ID}?contentDefinitionId=${CONTENT_DEFINITION_ID}`, {
        method: "DELETE",
        headers: HEADERS
    }).catch((exception) => {
        throw exception;
    });

    // Check for success
    if (RESPONSE.ok) {
        return RESPONSE.status;
    }
  	// There was an error
    await apiExceptionHandler(RESPONSE, "Deleting event instance failed");
}

File: nomad-samples/admin/py/event_instance_scheduler/instance/delete_instance.py
from constants.project_constants import *
from exceptions.api_exception_handler import *

import json, requests

def deleting_event_instance(AUTH_TOKEN: str, CONTENT_ID: str, CONTENT_DEFINITION_ID: str) -> dict:

    # Check for valid parameters
    if (not AUTH_TOKEN):
        raise Exception("Authentication Token: The authentication token is invalid")

    API_URL = f"{ADMIN_API_URL}/content/{CONTENT_ID}?contentDefinitionId={CONTENT_DEFINITION_ID}"
        
    # Create header for the request
    HEADERS = {
  	    "Authorization": "Bearer " + AUTH_TOKEN,
        "Content-Type": "application/json"
    }

    try:
        # Send the request
        RESPONSE = requests.delete(API_URL, headers= HEADERS)

        if not RESPONSE.ok:
            raise Exception()

    except:
        api_exception_handler(RESPONSE, "Deleting event instance failed")

